src/pages/ListView.js

```diff
@@ -5,18 +5,15 @@ import ListItem from '@/components/ListItem';
 import Components from '@/service/data/components.json';
 
 // Storing list objects
-const ListItems = Components.map( ( component ) => (
-    <ListItem component={component} key={component.id}  />
-  )
-)
+const ListItems = Components.map((component) => (
+  <ListItem component={component} key={component.id} />
+));
 
 // ListView Page
 const ListView = () => (
   <Page>
     <h1 className="mb-5">List of Common Components</h1>
-    <div>
-      {ListItems}
-    </div>
+    {ListItems}
```

Comment:
I feel ListItems needs to have a parent block element, like a container. What do you think?